src/System.Private.Interop/src/Shared/WindowsRuntimeMarshal.cs

```diff
@@ -977,8 +977,7 @@ internal class ReaderWriterLockTimedOutException : /*ApplicationException*/ Exce
             {
             }
 
-            /// I borrowed Vance's reader writer lock implementation from his blog as ReaderWriterLockSlim is
-            /// available in System.Core.dll!
```

Comment:
Meaning you want to delete the code or just the comment? I am happy to do the latter but would leave the former to you.